app/components/Combobox.tsx

```diff
@@ -68,12 +68,19 @@ export function Combobox<T extends Record<string, string | null | number>>({
     setLastSelectedOption(initialValue);
   }, [initialValue]);
 
+  // Reference for Fuse options: https://fusejs.io/api/options.html
+  const FUSE_OPTIONS = {
+    threshold: 0.42, // Empirically determined value to get an exact match for a Discord ID
+  };
+
   const filteredOptions = (() => {
     if (!query) return [];
 
     const fuse = new Fuse(options, {
+      ...FUSE_OPTIONS,
```

Comment:
Currently, these settings apply for *all* comboboxes used in the application. I can do a refactor later on to optionally pass a FUSE_OPTIONS prop from the parent, this is more for proof-of-concept

Reply:
Done the refactor